src/main/java/dk/kb/image/Facade.java

```diff
@@ -55,6 +61,12 @@ public static byte[] getGreyscale(BufferedImage img) throws IOException{
         return baos.toByteArray();
     }
 
+    /**
+     * Get the amount of unique colors from input image.
+     * @param img
+     * @return: Returns a string with the count of unique colors form the input image.
+     * @throws IOException
+     */
```

Comment:
I should have given you better pointers/explanation of Java Doc, sorry. Taking this method as an example:
* `Get the amount of unique colors from input image.`: This provides the extra information "unique", which is a good: It adds to the information from the method name.
* `@param img`: When there is nothing to write, don't state the param at all.
* `@return: Returns a string with the count of unique colors form the input image.`: The "Returns" is redundant and the recommended style (https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) is that "return" is implied, so `@return a string with the count of unique colors from the input image` would fit the style better.
* `@throws IOException`: Same comment as for `img`: Only state it if an explanation follows.